src/main/java/frc/robot/subsystems/AimingSubsystem.java

```diff
@@ -0,0 +1,283 @@
+  
+    mLeftElevatorMotor.setInverted(true);
+    mLeftWristMotor.setInverted(true);
+    mWristController.setTolerance(AimingConstants.WRIST_TOLERANCE_DEG);
+
+    //follower configuration
+    mRightWristMotor.follow(mLeftWristMotor);
+    mRightElevatorMotor.setControl(new Follower(mLeftElevatorMotor.getDeviceID(), false));
+  }
+
+  @Override
+  public void periodic() {
+    updateMotorModes();
+    elevatorPeriodic();
+    wristPeriodic();
+  }
+
+  private void elevatorPeriodic() {
+    //Clamping Rotation between domain
+    mDesiredElevatorDistanceIn = MathUtil.clamp(mDesiredElevatorDistanceIn, AimingConstants.MIN_ELEVATOR_DIST_IN, AimingConstants.MAX_ELEVATOR_DIST);
+    mCurrentElevatorDistanceIn = getCurrentElevatorDistance();
+
+    //Onboard PID
+    //mLeftElevatorMotor.setControl(elevatorVoltage.withPosition(mDesiredElevatorDistanceIn));
+
+    //Temp Regular PID
+    double elevatorPIDCalculation = mElevatorController.calculate(mCurrentElevatorDistanceIn, mDesiredElevatorDistanceIn);
+    mLeftElevatorMotor.set(elevatorPIDCalculation);
+  }
+
+  private void wristPeriodic() {
+    //Clamping Rotation between domain
+    mDesiredWristRotationDeg = MathUtil.clamp(mDesiredWristRotationDeg, AimingConstants.MIN_WRIST_ROTATION_DEG, AimingConstants.MAX_WRIST_ROTATION);
+    mCurrentWristRotationDeg = getCurrentWristRotation();
+
+    double offset = (Math.PI / 2);
+    double wristPIDCalculation = mWristController.calculate(mCurrentWristRotationDeg, mDesiredWristRotationDeg);    
+    double wristFFCalculation = mWristFeedforwardController.calculate(Math.toRadians(mDesiredWristRotationDeg) - offset, mLeftWristMotor.getEncoder().getVelocity() * AimingConstants.SPARK_THROUGHBORE_GEAR_RATIO);
+    mLeftWristMotor.set(wristPIDCalculation + wristFFCalculation);
+  }
+
+  private void updateMotorModes() {
+    SmartDashboard.updateValues();
+
+    AimingMotorMode mode = mChooser.getSelected();
+    
+    // Motors go towards setpoints
+    NeutralModeValue neutralmode;
+    IdleMode idleMode;
+
+    switch (mode) {
+        case BRAKE:
+            neutralmode = NeutralModeValue.Brake;
+            idleMode = IdleMode.kBrake;
+            break;
+        case COAST:
+            neutralmode = NeutralModeValue.Coast;
+            idleMode = IdleMode.kCoast;
+            break;
+        default:
+            neutralmode = NeutralModeValue.Brake;
+            idleMode = IdleMode.kBrake;
+            break;
+    }
+
+    mLeftElevatorMotor.setNeutralMode(neutralmode);
+    mRightElevatorMotor.setNeutralMode(neutralmode);
+
+    mRightWristMotor.setIdleMode(idleMode);
+    mLeftWristMotor.setIdleMode(idleMode);
+  }
+
+  // Contains Smart Dashboard Statements ONLY ON DEBUG
+  private void debugSmartDashboard() {
+    if (CompConstants.DEBUG_MODE) {
+      SmartDashboard.putNumber("Current Wrist Rotation", mCurrentWristRotationDeg);
+      SmartDashboard.putNumber("Current Elevator Distance", mCurrentElevatorDistanceIn);
```

Comment:
Add Desired Wrist Rotation, Raw Encoder Positions, and potentially atSetpoints functions.

Reply:
Added the debug statements.